docs/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Change logging of celery and django to filesystem (OSIDB-418)
 - Implement validation for CWE ID chain in a Flaw (OSIDB-357)
+- Implement validation for embargoed flaws not be able to have public trackers (OSIDB-350)
+
+## [2.3.4] - 2022-12-15
+### Changed
+- Change logging of celery and django to filesystem (OSIDB-418)
+- Implement validation for CWE ID chain in a Flaw (OSIDB-357)
```

Comment:
should be removed, probably came from a conflict

Reply:
this is true, updated to reflect current state.